pallets/did/src/utils.rs

```diff
@@ -16,17 +16,45 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
+use crate::{
+	did_details::{DidCreationDetails, DidPublicKey},
+	BalanceOf, Config, KeyIdOf,
+};
 use parity_scale_codec::Encode;
-use sp_runtime::traits::Hash;
+use sp_core::Get;
+use sp_runtime::traits::{Hash, Zero};
 use sp_std::vec::Vec;
 
-use crate::{did_details::DidPublicKey, Config, KeyIdOf};
```

Comment:
We usually keep the internal imports at the bottom, separated from the rest by a newline. I think we should keep that.

Reply:
May I ask what kind of formatter do you use?

Reply:
This is not a formatter rule, it's just a convention we use. Crate imports always come after external imports, and to avoid the formatter to shuffle them around alphabetically, we put a new line.